testcontainers-bom/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+    <groupId>com.playtika.testcontainers</groupId>
+    <artifactId>testcontainers-bom</artifactId>
+    <version>1.55-SNAPSHOT</version>
+
+    <properties>
+        <revison>1.55-SNAPSHOT</revison>
```

Comment:
you need to link to parent, otherwise release job will not touch this artifact

Reply:
OK, thank you for your advice